IHP/View/CSSFramework.hs

```diff
@@ -35,6 +35,10 @@ instance Default CSSFramework where
                         ErrorFlashMessage message -> [hsx|<div>{message}</div>|]
                 , styledFlashMessages
                 , styledFormField
+                , styledTextFormField
+                , styledCheckboxFormField
+                , styledSelectFormField
```

Comment:
@mpscholten I'd like to check the `selectField` . Is there an example somewhere on how to populate an Int field with custom input. I'm looking for something like `selectField #number [(1, "one"), (2, "two")]` ?

Reply:
If you follow https://ihp.digitallyinduced.com/Guide/form.html#select-inputs you should be able to get it working with this instance:

```haskell
instance CanSelect (Int, Text) where
    type SelectValue (Int, Text) =  Int
    selectValue (int, text) = int
    selectLabel (int, text) = text
```

Reply:
Thanks. I'll create a follow up PR to add it to the docs